manifests/installation-script

```diff
@@ -266,7 +270,6 @@ log("setup grafana");
 if !hasgrafana {
 	createOrgScript = shebang + `
 	sleep 50
-	sleep 50
```

Comment:
Add this, this delay is introduced for delaying grafana curl

Reply:
Done, added Sleep for the fix